asmcli/lib/parse.sh

```diff
@@ -284,6 +288,10 @@ x_parse_install_args() {
         context_set-option "MANAGED" 1
         shift 1
         ;;
+      --legacy)
+        context_set-option "LEGACY" 1
+        shift 1
+        ;;
```

Comment:
I don't think we need to enable client-side installs for `x install`. In fact we might want to just print a warning that `x install` is included in `--managed` now.

Reply:
Done.